tools/detection2d/sample/yolox-s+-opt-T4-960x960.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+# Copyright (c) Megvii, Inc. and its affiliates.
+
+# from https://drive.google.com/drive/folders/1A7-4oSJCtu_9lFQoAKZUkwxYAqGvzAny
```

Comment:
Please remove internal data for OSS users.
If you can use onnx file from OSS projects, please write that link.
If you use internal model, please upload to S3 model registry as https://github.com/tier4/AWML/blob/main/docs/design/architecture_s3.md.